pandas/tests/indexes/multi/test_constructor.py

```diff
@@ -472,3 +472,83 @@ def test_from_tuples_with_tuple_label():
     idx = pd.MultiIndex.from_tuples([(2, 1), (4, (1, 2))], names=('a', 'b'))
     result = pd.DataFrame([2, 3], columns=['c'], index=idx)
     tm.assert_frame_equal(expected, result)
+
```

Comment:
can you add something like before this section (and do the same for the other big sections)

```
    # ------------------------------------------------------------------------
    # from_frame
    # ------------------------------------------------------------------------
 ```